x-pack/plugins/cases/public/components/user_actions/comment/show_alert_table_link.tsx

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Could you please add a test for this component?

Reply:
@cnasikas I was going to do it., but it is literally just a link. And the link is validated in the build tests.

Reply:
The test I have in mind is to test that the `navigateToCaseView` was called with the correct parameters.

Reply:
ok...